### DISCUSSION TITLE
Issue in workflow Informatics for RNA-seq

### DISCUSSION THREAD

**Discussion by PabloJGiraudi** [biostars/biostar-handbook-issues#220] [General]
Hello,

I try to run the RNA-Seq workflow on two different computers (one using ubuntu in a laptop with linux and the second time in a computer with Windows using Linux Ubuntu Subsystem). Unfortunately, when running the workflow I obtain the following errors in the second step (Unpack and index).

The workflow stops when trying to index reference genomes for salmon.

command:

    make -f flow/rnaseq.mk unpack index

```
Total time for call to driver() for forward index: 00:00:33
# Index: idx/Homo_sapiens.GRCh38.dna.chromosome.22.fa
make -s -f src/salmon.mk CDS=refs/Homo_sapiens.GRCh38.cds.all.fa index
make[1]: salmon: No such file or directory
make[1]: *** [src/salmon.mk:59: idx/Homo_sapiens.GRCh38.cds.all.fa] Error 127
make: *** [flow/rnaseq.mk:54: index] Error 2
```
(bioinfo)

Could be a problems of my computer setup or it is a problem in the workflow? I have executed the command as described in the Informatics for RNA-seq workflow guide?

**Comment by ialbert** [comment]
make sure salmon is installed 

    mamba install salmon

**Reply by PabloJGiraudi** [reply]
many thanks, Albert,  after installing salmon and additional software as described in your guideline, all workflow runs well!!!

Pabllo
